src/main/java/org/mskcc/kickoff/lims/SampleInfoImpact.java

```diff
@@ -1058,6 +1054,16 @@ private void populatePoolNormals(User apiUser, DataRecordManager dataRecordManag
         }
     }
 
+    private List<DataRecord> getPotentialPooledNormalQCs(User apiUser, DataRecordManager dataRecordManager) throws Exception {
+        String query = String.format("%s LIKE '%s'", VeloxConstants.OTHER_SAMPLE_ID,
```

Comment:
string has two %s, but three arguments, do we need to remove one?

Reply:
yep, it can be removed, I just copied that code from another place, I'll remove it